### ISSUE TITLE
Gripper turns over in place object

### ISSUE THREAD

**Issue by pazeshun** [start-jsk/jsk_apc#2213]
Please see first picking of https://drive.google.com/open?id=0BxxBA3J-CunGUnlLM0g2WjZFNVk

**Comment by pazeshun** [comment]
```
(send *ti* :move-arm-body->bin-overlook-pose :rarm :c)
```
![screenshot from 2017-07-04 19 11 31](https://user-images.githubusercontent.com/14994939/27826067-b35dd1d2-60ec-11e7-978d-3783ccfeba19.png)
->
```
(send *ti* :ik->cardboard-center :rarm :a :offset #f(0 0 200) :rotation-axis :z :use-gripper t)
```
![screenshot from 2017-07-04 19 11 14](https://user-images.githubusercontent.com/14994939/27826079-c20fd2d4-60ec-11e7-8111-0160012aa21f.png)

**Comment by pazeshun** [comment]
Perhaps we should add `:additional-check` to cardboard IK

**Comment by pazeshun** [comment]
Fixed in #2216

**Comment by wkentaro** [comment]
Can I close this? Or you postpone this to 3.3.0?

**Comment by wkentaro** [comment]
Please reopen unless it is fixed.

**Comment by pazeshun** [comment]
Sorry. This is fixed in #2203
